jib-core/src/main/java/com/google/cloud/tools/jib/event/events/LogEvent.java

```diff
@@ -89,4 +91,24 @@ public Level getLevel() {
   public String getMessage() {
     return message;
   }
+
+  @VisibleForTesting
+  @Override
+  public boolean equals(Object obj) {
+    if (obj == this) {
+      return true;
+    }
+    if (!(obj instanceof LogEvent)) {
+      return false;
+    }
+
+    LogEvent other = (LogEvent) obj;
+    return level == other.getLevel() && message.equals(other.getMessage());
```

Comment:
nit: can use the private fields directly since checking for equality is checking on the contents rather than the return values of getters.